streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfo.java

```diff
@@ -150,9 +159,17 @@ public ByteBuffer encode() {
                     encodePartitionsByHostAsDictionary(out);
                     out.writeInt(errCode);
                     break;
+                case 6:
+                    out.writeInt(usedVersion);
+                    out.writeInt(commonlySupportedVersion);
+                    encodeActiveAndStandbyTaskAssignment(out);
+                    encodePartitionsByHostAsDictionary(out);
```

Comment:
Can we can encode the topicname index, just once instead of duplicating once for active and standby. wdyt? I can take up this change.

Reply:
Sure.